src/content/docs/dev/send-tokens/interchain-tokens/integrate-custom-token.mdx

```diff
@@ -92,32 +129,30 @@ An Interchain Token is an ERC-20 that is connected to [ITS](https://github.com/a
   Your browser does not support SVG
 </object>
 
-Use the `interchainTransfer` function to send tokens across chains:
+Use the [`interchainTransfer`](https://github.com/axelarnetwork/interchain-token-service/blob/94677f1072da127a0ee7ff5ab6784dcfe11789e8/contracts/InterchainTokenService.sol#L559) function to send tokens across chains.
```

Comment:
what is this branch we're linking to?

Reply:
This is the main branch. Copying the permalink automatically directs to the commit.